list/letters-filter/style/letters-filter.scss

```diff
@@ -0,0 +1,21 @@
+
+
+[data-focus='letters-filter-bar'] {
+ /* float: left;
+  margin-right: 30px;*/
+  [data-focus='letter-element'] {
+    display: inline-block;
+    padding: 0 10px 0 10px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 10px` instead of `0 10px 0 10px`